inc/components/components/post-content/component.json

```diff
@@ -3,6 +3,28 @@
   "_alias": "irving/text",
   "category": "Post",
   "description": "Get the post content.",
+  "config": {
+    "content": {
+      "default": "",
+      "type": "string"
+    },
+    "html": {
+      "default": true,
+      "type": "bool"
+    },
+    "oembed": {
+      "default": true,
+      "type": "bool"
+    },
+    "style": {
+      "default": [],
+      "type": "array"
+    },
+    "tag": {
+      "default": "span",
```

Comment:
Why not a `div` for the default?

Reply:
The default for text is `span`, so I copied that over, but `div` makes sense to me too.